playpen/coverage/coverage_hook

```diff
@@ -57,6 +60,9 @@ report will fail if coverage has not yet been installed
 """
 _root_epilog = "This script is intended to be run with root privileges."
 
+_DISTRIBUTION = platform.dist()[0]
+_DISTRIBUTION_MAJOR_VERSION = platform.dist()[1].split('.')[0]
```

Comment:
We've had problems with this not working in f24, so I think it'd be good to check the `platform.dist` returns are sane and fail quickly here if not.